actions/workflows/st2_e2e_tests.yaml

```diff
@@ -138,105 +141,117 @@ workflows:
-                    - test_quickstart_local_script
+                # on-success:
+                #     - test_quickstart_local_script
             test_quickstart_local_script:
                 action: core.remote
                 input:
                     hosts: <% $.host %>
                     env: <% $.env %>
                     cmd: st2 run tests.test_quickstart_local_script_actions <% $.st2_cli_args %>
                     timeout: 600
-                on-success:
-                    - test_quickstart_remote_script
+                # on-success:
+                #     - test_quickstart_remote_script
             test_quickstart_remote_script:
                 action: core.remote
                 input:
                     hosts: <% $.host %>
                     env: <% $.env %>
                     cmd: st2 run tests.test_quickstart_remote_script_actions <% $.st2_cli_args %>
                     timeout: 600
-                on-success:
-                    - test_quickstart_passive_sensor
+                # on-success:
+                #     - test_quickstart_passive_sensor
             test_quickstart_passive_sensor:
                 action: core.remote
                 input:
                     hosts: <% $.host %>
                     env: <% $.env %>
                     cmd: st2 run tests.test_quickstart_passive_sensor <% $.st2_cli_args %>
                     timeout: 600
-                on-success:
-                    - test_quickstart_polling_sensor
+                # on-success:
+                #     - test_quickstart_polling_sensor
             test_quickstart_polling_sensor:
                 action: core.remote
                 input:
                     hosts: <% $.host %>
                     env: <% $.env %>
                     cmd: st2 run tests.test_quickstart_polling_sensor <% $.st2_cli_args %>
                     timeout: 600
-                on-success:
-                    - test_quickstart_python
+                # on-success:
+                #     - test_quickstart_python
             test_quickstart_python:
                 action: core.remote
                 input:
                     hosts: <% $.host %>
                     env: <% $.env %>
                     cmd: st2 run tests.test_quickstart_python_actions <% $.st2_cli_args %>
                     timeout: 600
-                on-success:
-                    # - test_quickstart_key_triggers
-                    - test_quickstart_trace
-            # test_quickstart_key_triggers:
-            #     action: core.remote
-            #     input:
-            #         hosts: <% $.host %>
-            #         env: <% $.env %>
-            #         cmd: st2 run tests.test_key_triggers <% $.st2_cli_args %>
-            #         timeout: 600
-            #     on-success:
-            #         - test_quickstart_trace
+                # on-success:
+                #     - test_quickstart_key_triggers
+            test_quickstart_key_triggers:
```

Comment:
Reverting https://github.com/StackStorm/st2cd/pull/210 as the fix is there for keystore scope `st2kv.system.` https://github.com/StackStorm/st2tests/pull/67  and st2tests has branching as well: https://github.com/StackStorm/st2cd/pull/211